mxcube3/ui/actions/general.js

```diff
@@ -100,6 +108,14 @@ export function getInitialStatus() {
         'Content-type': 'application/json'
       }
     });
+    const beamlineLimits = fetch('mxcube/api/v0.1/beamline', {
```

Comment:
As far as I understand it, this is not only getting limits but the whole dictionary representation of the `BeamlineSetupMediator` object. So maybe a more appropriate variable name like `beamlineSetup` would be better ?